templates/_helpers.tpl

```diff
@@ -522,6 +522,9 @@ Params:
 {{- end -}}
 
```

Comment:
Would be useful to give comments for other fucntions, similar to earlier functions. (about function, usage and params). Was not clear about the params and their purpose. all the params are being passed. expected only source/sourceEnabled and default params. Purpose of other params was not clear.